### ISSUE TITLE
T1 L5.18: "Log-loss error function": does not define log loss formula

### ISSUE THREAD

**Issue by e7dud7e** [udacity/sdc-issue-reports#1331]
The video in L5.18 "log-loss error function" page ends with Luis saying "how do we define this error function? That's what we'll do next."  The next lesson 5.19 is "discrete vs. continuous".  The first time I see the log loss error function is in L 5.23 "gradient descent."  There, in the text it says "And now let's recall ... the error formula is..." [and it gives the log loss function, but doesn't call it that.]  I'm guessing there's a lecture that was created for another nanodegree that is missing in the carnd?

**Comment by baumanab** [comment]
Thanks for reporting this. I actually think this is a case where it was intended to put certain content after that video but it was later determined to be best elsewhere.  We will put a note and link in the classroom.

**Comment by baumanab** [comment]
A note has been added to the classroom.
